docs/developer-docs/latest/getting-started/quick-start.md

```diff
@@ -87,9 +87,15 @@ canonicalUrl: https://docs.strapi.io/developer-docs/latest/getting-started/quick
 Strapi offers a lot of flexibility. Whether you want to go fast and quickly see the final result, or would rather dive deeper into the product, we got you covered.
 
 ::: prerequisites
-Make sure [Node.js and npm are properly installed](/developer-docs/latest/setup-deployment-guides/installation/cli.md#preparing-the-installation) on your machine. It is also possible to use Yarn instead of npm (see [install the Yarn package](https://yarnpkg.com/en/)).
+
+The installation requires software prerequisites to be already installed on your computer:
```

Comment:
```suggestion
The installation requires the following software to be already installed on your computer:
```

I would suggest this wording (because "prerequisites" is already mentioned in the callout title, and because "the following …:" is how we write it in other parts of the docs).

Reply:
Sounds good to me. I will apply this suggestion first thing tomorrow morning!

Reply:
Awesome! So I'll merge this PR and it will be deployed next Wednesday 🚀